libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -50,11 +50,15 @@ export type WalletType = 'injected' | 'jsonRpc' | 'view' | 'snap';
 export interface VegaConnectDialogProps {
   connectors: Connectors;
   riskMessage?: ReactNode;
+  contentOnly?: boolean;
+  onClose?: () => void;
 }
 
 export const VegaConnectDialog = ({
   connectors,
   riskMessage,
+  contentOnly,
```

Comment:
I don't like this approach. Its weird to me that you pass a prop to unwrap the component. Lets not change now though

Reply:
Why not just import and use the inner child component directly. This is unnecessary logic